app/views/uploads/new.html.erb

```diff
@@ -1,226 +1,136 @@
+    <div class="fileinput fileinput-new input-group" style="width:50%" data-provides="fileinput">
+      <div class="form-control" data-trigger="fileinput"><i class="glyphicon glyphicon-file fileinput-exists"></i> <span class="fileinput-filename"></span></div>
+      <label for="inputFile" class="sr-only">QRDA File</label>
+      <span class="input-group-addon btn btn-default btn-file btn-info">
+      <span class="fileinput-new"><i class="fa fa-fw fa-mouse-pointer"></i> Select file</span>
+      <span class="fileinput-exists"><i class="fa fa-fw fa-refresh"></i> Change</span>
+        <input type="file" name="file" id="inputFile" required accept=".xml,.zip">
+      </span>
+      <a href="#" class="input-group-addon btn btn-warning fileinput-exists" data-dismiss="fileinput">Remove</a>
+    </div>
+  </fieldset>
+
+  <fieldset>
+    <legend class="control-label">Annual Update Bundle</legend>
+    <div class="form-group" id="radioYearContainer">
+      <% if BUNDLES['2016'] %>
+      <div class="radio">
+      <label class="radio">
+        <input required class="proyear" type="radio" name="year" value="2016" id="2016"> 2015 Bundle for the 2016 Reporting Period
+      </label>
+      </div>
+      <% end %>
+      <% if BUNDLES['2017'] %>
+      <div class="radio">
+      <label class="radio">
+        <input required class="proyear" type="radio" name="year" value="2017" id="2017"> 2016 Bundle for the 2017 Reporting Period
+      </label>
+      </div>
+      <% end %>
+      <% if BUNDLES['2018'] %>
+      <div class="radio">
+      <label class="radio">
+        <input required class="proyear" type="radio" name="year" value="2018" id="2018"> 2017 Bundle for the 2018 Reporting Period
+      </label>
+      </div>
+      <% end %>
+    </div>
+    <% unless BUNDLES['2016'] %>
+      <span id="bundle-message" class="help-block">To enable the <em>2015 Bundle for the 2016 Reporting Period</em>, please install a 2015 Cypress bundle</span>
+    <% end %>
+    <% unless BUNDLES['2017'] %>
+      <span id="bundle-message" class="help-block">To enable the <em>2016 Bundle for the 2017 Reporting Period</em>, please install a 2016 Cypress bundle</span>
+    <% end %>
+    <% unless BUNDLES['2018'] %>
+      <span id="bundle-message" class="help-block">To enable the <em>2017 Bundle for the 2018 Reporting Period</em>, please install a 2017 Cypress bundle</span>
+    <% end %>
+  </fieldset>
+  <hr>
+  <fieldset>
+    <legend class="control-label">CMS Reporting Program</legend>
+    <% @program_years.each do |program_year| %>
+      <div class="form-group program-radio" id="<%= "radioContainer_#{program_year[0]}" %>" style="display: none;">
+          <%= render partial: 'program_radio', locals: {programs: program_year} %>
+      </div>
+    <% end %>
+  </fieldset>
+  <hr>
+  <fieldset>
+    <legend class="control-label">Document Type</legend>
+    <div class="form-group" id="radioDocumentContainer">
+    <% @document_years.each do |document_year| %>
```

Comment:
Maybe I'm dense but it took me a while to figure out this loop and the one inside of it. Would you mind changing the iteration to using `|key, value|` syntax?
something like:
`<% @document_years.each do |year, document_types| %>`

and then something like this for the inner loop
`<% document_types.each do |category, specifications| %>`